recipes/libxml2/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+if [[ $(uname) == 'Darwin' ]]; then
+  export LIBRARY_SEARCH_VAR=DYLD_FALLBACK_LIBRARY_PATH
+  OPTS="--without-python"
+elif [[ $(uname) == 'Linux' ]]; then
+  export LIBRARY_SEARCH_VAR=LD_LIBRARY_PATH
+  OPTS="--with-python="${PREFIX}""
```

Comment:
Maybe we should just disable it across the board. This might get a bit confusing. Also, the more I think about it the more I would like to just use `lxml` instead.